src/encoded/tests/test_audit_antibody_lot.py

```diff
@@ -98,4 +98,3 @@ def test_audit_control_characterizations(testapp, antibody_lot, base_target1):
     errors = res.json['audit']
     print(errors)
     assert 'NOT_COMPLIANT' not in errors
-
```

Comment:
Did you remove this empty line on purpose or did your editor do it automatically?